spec/requests/v1/cors_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe "the requests support CORS headers" do
+  it 'Returns the response CORS headers' do
+
+    hello = Phrase.create(expression: "Hello",
+                    image: "blahblahblah.url",
+                    )
+
+    post '/v1/graphql', params: { query: query(id: hello.id)}, headers: {'ORIGIN' => '*'}
```

Comment:
This is basically the same test as in our phrase query spec, just needed something to make sure the correct header was coming through.

Reply:
Great testing here though, targets exactly what we need it to

Reply:
Nice work making a test for this